FFXIVTheMovie/FFXIVTheMovie/ParserV3/Parser.cs

```diff
@@ -85,7 +85,7 @@ public void GenerateQuestScript()
             outputCpp.Add("{");
             outputCpp.Add("public:");
             outputCpp.Add($"  {questId}() : Sapphire::ScriptAPI::EventScript( {questNumber} ){{}}; ");
-            outputCpp.Add($"  ~{questId}() = default; ");
+            outputCpp.Add($"  ~{questId}() {{}} ");
```

Comment:
What does this do? I think the default destructor is fine.

Reply:
It's a habit, in truth it doesn't change anything but I can't see it with default 🙈

Reply:
In anycase, if this PR doesn't need you can close this